docs/editors/overview.md

```diff
@@ -154,20 +169,14 @@ from source.
 **✅**: it is possible to import a build such as an sbt project directly from
 the editor.
 
-**Requires browser**: importing a build requires additional steps in a web
-browser using a localhost server. It is not possible to import a build within
-the editor.
+**Manual import**: importing a build requires running `bloopInstall` in one of
```

Comment:
Does eclipse LSP support executing commands? If so, running "Import build" should do the trick.

The "Requires browser" option also works for Eclipse as long as the `-Dmetals.http=true` is enabled. I believe this is a better workflow than manually running "sbt bloopInstall" because Metals "import build" automatically enables downloading of dependency sources + SemanticDB.

Reply:
I haven't found any way to run commands, so probably running from localhost makes sense. I will revert and add the option to the plugin.

Reply:
What do you mean by running commands? Do you mean, like running metals from eclipse?

Reply:
The LSP `workspace/executeCommand` request. Here's the list of commands that Metals supports http://scalameta.org/metals/docs/editors/new-editor.html#metals-server-commands

Reply:
ok, thank you, I'll check that out